.gitlab-ci.yml

```diff
@@ -44,7 +44,15 @@ analysis:licenses:
   stage: analysis
   script:
     - git fetch --depth=1 origin master
-    - ./gradlew :dd-sdk-android:checkThirdPartyLicences
+    - ./gradlew :dd-sdk-android:checkThirdPartyLicences :dd-sdk-android-timber:checkThirdPartyLicences
+
+analysis:api-surface:
+  tags: [ "runner:docker", "size:large" ]
```

Comment:
What does `size:large` do in terms of picking the right runner?

Reply:
Not sure on the specifics, but the idea is that it will spawn the container in a runner that is made for large tasks.